tests/rkt_net_test.go

```diff
@@ -381,7 +381,7 @@ func TestNetTemplates(t *testing.T) {
 		Type: "ptp",
 		Ipam: ipamTemplateT{
 			Type:   "host-local",
-			Subnet: "10.1.3.0/24",
+			Subnet: "11.11.3.0/24",
```

Comment:
11.* is assigned to [USDOD](https://en.wikipedia.org/wiki/List_of_assigned_/8_IPv4_address_blocks#List_of_assigned_.2F8_blocks_to_the_United_States_Department_of_Defense).